### ISSUE TITLE
Wrong 'da' translation of "Password Reset Email" /locale/da/LC_MESSAGES /django.po

### ISSUE THREAD

**Issue by andreassh** [pennersr/django-allauth#3701]
I noticed a wrong and misleading translation of "Password Reset Email" in file /locale/da/LC_MESSAGES /django.po

This is the snippet
```
#: templates/account/email/password_reset_key_subject.txt:3
#: templates/account/email/unknown_account_subject.txt:3
msgid "Password Reset Email"
msgstr "Token for nulstilling af adgangskode var ugyldig"
```

 "Token for nulstilling af adgangskode var ugyldig" in English means "Token for resetting password was invalid", leading me to believe something in the django validation was off. Took me a while to debug what was happening. Turns out it's just the translation that i off.

Should say something like.

```
#: templates/account/email/password_reset_key_subject.txt:3
#: templates/account/email/unknown_account_subject.txt:3
msgid "Password Reset Email"
msgstr "Nulstilling af Password Email"
```

**Comment by pennersr** [comment]
Closed via d29fd173

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3701
